src/main/java/com/nha/abdm/wrapper/hip/hrp/link/userInitiated/LinkService.java

```diff
@@ -86,14 +98,26 @@ public void onInit(InitResponse initResponse) {
 
     log.info("onInit body : " + onInitRequest.toString());
     try {
-      ResponseEntity<GenericResponse> responseEntity =
-          requestManager.fetchResponseFromGateway(onInitLinkPath, onInitRequest);
-      log.info(onInitLinkPath + " : onInitCall: " + responseEntity.getStatusCode());
+      log.info("Sending otp request to HIP");
+      GenericRequest genericRequest=GenericRequest.builder().abhaAddress(initResponse.getPatient().getId()).patientReference(initResponse.getPatient().getReferenceNumber()).build();
```

Comment:
can we rename this request class to RequestOTP since it is doing that job. if you are intending to use the same class attributes for some other job then we can have two different classes with same attributes, its perfectly fine.